src/Application/Commands/CopyExistingPersonRecordsToMatchbot.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace BigGive\Identity\Application\Commands;
+
+use BigGive\Identity\Application\Auth\Token;
+use BigGive\Identity\Repository\PersonRepository;
+use Doctrine\DBAL\Connection;
+use Monolog\DateTimeImmutable;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\RoutableMessageBus;
+
+/**
+ * Command only needed temporarily, to send copies of all Person records that were last updated before
+ * we started a live sync to matchbot. Once the job is complete this can be deleted.
+ */
+#[AsCommand(name: 'identity:copy-existing-person-records-to-matchbot')]
```

Comment:
I do find it easier to have a Composer shortcut command to quickly work out what args to use when invoking from ECS, where the syntax to launch Docker filesystem binaries requires remembering an internal absolute path, but not a blocker

Reply:
ok - I wasn't expecting to need to launch it from ECS though. The `delete-unusable-person-records` is run directly from our terraform setup without going through composer.

Reply:
Yep, I commented here before I saw the infrastructure part and missed that earlier comment, makes sense